plone/app/querystring/profiles/default/registry.xml

```diff
@@ -795,6 +803,7 @@
     <value key="sortable">False</value>
     <value key="operations">
       <element>plone.app.querystring.operation.string.contains</element>
+      <element>plone.app.querystring.operation.string.search</element>
```

Comment:
So if an editor is configuring a search block and adds a criterion with the SearchableText index, they'll have to pick between "Contains" or "Search" as the operation. That seems like bad UX since it's not clear why to choose one or the other.

If you need this for integration with elasticsearch, wouldn't it make more sense for your elasticsearch integration package or your policy package to add this operation (and probably remove the other one)?